synfig-studio/src/gui/dialogs/dialog_color.cpp

```diff
@@ -65,9 +65,9 @@ Dialog_Color::Dialog_Color():
 
 	create_color_edit_widget();
 	create_close_button();
-	create_set_color_button("synfig-set_outline_color", _("Set as Outline"), 0,
+	create_set_color_button("set_outline_color", _("Set as Outline"), 0,
 			sigc::mem_fun(*this, &Dialog_Color::on_set_oc_pressed));
-	create_set_color_button("synfig-set_fill_color", _("Set as Fill"), 1,
+	create_set_color_button("set_fill_color", _("Set as Fill"), 1,
```

Comment:
these icons don't follow the filename pattern: do not end with "_icon" :/
Open an issue?

Reply:
I think yes. Better to fix this.

Reply:
In this PR or another one?

Reply:
Added here. I can transpose to another PR if you think it is better.